pkg/globalregistry/reconciler/replicationrulestatus.go

```diff
@@ -94,8 +94,7 @@ func (ra *rRuleRemoveAction) Perform(ctx context.Context, reg globalregistry.Reg
 	for _, rRule := range rRules {
 		destructibleReplicationRule, ok := rRule.(globalregistry.DestructibleReplicationRule)
 		if !ok {
-			// TODO: error handling
-			continue
+			panic("replication rule cannot be deleted")
```

Comment:
Is this logic already used in the k8s controller? If so, I would prefer to not panic in the controller..

Reply:
I agree with you. Fixed.